integration-tests/load-tests/signatureRegistry.test.ts

```diff
@@ -0,0 +1,160 @@
+import "@frequency-chain/api-augment";
```

Comment:
This is the actual load test. Other changes to integration test files are just to support this.